docs/docs/cmd/spo/homesite/homesite-set.mdx

```diff
@@ -16,10 +16,22 @@ m365 spo homesite set [options]
 
 ```md definition-list
 `-u, --siteUrl <siteUrl>`
-: The URL of the site to set as Home site.
+: The URL of the home site
+
+`--draftMode [draftMode]` 
+: Set draft status. Valid values are: `true`, `false`.
+
+`--audienceIds [audienceIds]` 
+: Comma-separated list of group Ids to set as audiences. Specify either `audienceIds` or `audienceNames`, but not both.
+
+`--audienceNames [audienceNames]` 
+: Comma-separated list of group names to set as audiences. You must use either `audienceIds` or `audienceNames`, but not both. Pass '' to clear audience.
+
+`--targetedLicenseType [targetedLicenseType]` 
+: Value of either `everyone`, `frontLineWorkers` or `informationWorkers`.
```

Comment:
We're missing some kind of description here what the option is about. For example: `Create an experience for a distinct audience`